extensions/hadoop/src/test/java/com/hazelcast/jet/hadoop/file/TextFileFormatTest.java

```diff
@@ -35,10 +35,7 @@ public void readTextFileAsSingleItem() {
                                                       .glob("file.txt")
                                                       .format(FileFormat.text());
 
-        assertItemsInSource(source,
-                "Text contents of" + System.lineSeparator() +
-                        "the file." + System.lineSeparator()
-        );
+        assertItemsInSource(source, "Text contents of\nthe file.\n");
```

Comment:
This is what I had originally, but didn't pass on Windows due to different line endings. I think it depends on `git config core.autocrlf` value. Isn't setting this config to true a standard practice on windows? It results in the file having the following content:
`Text contents of\r\nthe file.\r\n`

Reply:
I don't know, but we shouldn't depend on some git setting. It's typically better to not do any conversion. Today even Windows Notepad can open files with linux file endings. I have that option off.

I changed the text to load the file using `Files.readAllBytes` so that it can work regardless of that setting.